src/app/debug/table/table.component.ts

```diff
@@ -540,7 +597,7 @@ export class TableComponent implements OnInit, OnDestroy {
       for (let report of data) {
         const reportData: ReportData = {
           report: report,
-          currentView: this.viewSettings.currentView,
+          currentView: this.viewSettings.currentView ?? ({} as View),
```

Comment:
Setting it to an empty object if it's null should not be the solution here.

Reply:
Youre totally right, i changed it to check if currentView is defined before  trying to set it to the currentView